vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -37,15 +37,14 @@ class Graph(VizroBaseModel):
     """
 
     type: Literal["graph"] = "graph"
-    figure: CapturedCallable = Field(..., import_path=px)
+    figure: CapturedCallable = Field(..., import_path=px, mode="graph")
```

Comment:
Should we add the `description` here as we did for the other Vizro figure-like object?

Reply:
Yes, these were very inconsistent before actually! I've just fixed it: [d452ee4](https://github.com/mckinsey/vizro/pull/547/commits/d452ee475f2cd050b1269280efc647fb1b1eeb6e).